frontend/public/components/dashboard/utilization-card/utilization-card.scss

```
@@ -1,51 +1,55 @@
-.co-utilization-card__area-chart {
-  margin: 0;
-  padding: 0;
-}
-
-.co-utilization-card__axis {
-  padding: 0 0.5rem;
+.co-utilization-card__item .co-u-hidden {
+  display: none;
 }
 
-.co-utilization-card__item {
-  border-bottom: 1px solid $pf-color-black-300;
-  font-size: 1.333em;
-  margin: 0;
+.co-utilization-card__body {
   padding: 0;
 }
-
-.co-utilization-card__item-chart {
-  padding: 0 0.5rem;
-  text-align: center;
+.co-utilization-card__item:not(:first-child) {
+  border-top: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
 }
-
-.co-utilization-card__item-chart--narrow {
-  text-align: center;
+.co-utilization-card__item > * {
+  padding: var(--pf-global--spacer--md);
 }
-
-.co-utilization-card__item-chart--wide {
-  border-left: 1px solid $pf-color-black-300;
+.co-utilization-card__item__text {
+  color: var(--pf-global--Color--200);
+  font-size: var(--pf-global--FontSize--sm);
 }
-
-.co-utilization-card__item-current {
-  padding-right: var(--pf-c-card--child--PaddingRight);
+.co-utilization-card__item .pf-c-title, .co-utilization-card__item .pf-l-flex > * {
```

```suggestion
.co-utilization-card__item .pf-c-title {
```
This can be simplified now too.